docs/pages/deploy/customize-prebuilt.zh-TW.mdx

```diff
@@ -10,7 +10,7 @@ import { Callout } from 'nextra-theme-docs'
 
 雖然 Zeabur 已經涵蓋眾多由社群提供的 Prebuilt 範本（Prebuilt Marketplace），
 你可能會需要部署自己的 Docker image、配置 Stateful 的服務（儲存空間），或者是
-使用更進階的環境變數功能（如曝露 expose 某個變數給其他服務）。
+使用更進階的環境變數功能（如曝露某個變數給其他服務）。
```

Comment:
```suggestion
使用更進階的環境變數功能（如暴露某個變數給其他服務）。
```

according to https://github.com/zeabur/zeabur/pull/331/files#diff-b92d6ef3d2422b5c06b7d90ab72cc6a828e866d78a612cc6d4d30ac6a4e98108R22

Reply:
Note: Both are used in the original revision, I just leave it.